crates/astria-core/src/sequencer/v1/asset.rs

```diff
@@ -119,6 +122,7 @@ impl From<String> for Denom {
 
 /// Asset ID, which is the hash of the denomination trace.
 #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize))]
```

Comment:
This is very likely not going to do what you want. Have a look at `RollupId`. `astria_core::serde` provides a utility for deriving a serde impl that serialises bytes as base64